.github/pull_request_template.md

```diff
@@ -1,21 +1,38 @@
 ---
 name: Pull Request
-about: A simple pull request template.
-issue: Issue # if applicable
+about: Request to pull code changes into the repository.
+title: ''
+assignees: ''
 
 ---
 
-Provide a simple description of what the PR achieves. 
+## Relevant Issues
```

Comment:
I don't think we should start with relevant issues, seems better to put this at the bottom of the template imo. That applies to all the templates. It's supplemental information, the meat of the PR should be the users own description of what it does, so we should lead with that.

Reply:
I'll adjust it, but I don't really agree. If this repo was bot or site, I wouldn't even suggest a special section for it at all as most of us know to link the issue, but with beginners being a focus on seasonalbot, I put it at the top; It's commonly forgotten, it's important context to the PR and it's easy to consistently see across multiple PRs this way.

Reply:
I agree with @scragly. Issues can provide important context. If I see a PR I probably want to know what issue it's tackling first before I even look at the changes. The section doesn't take up much space so it won't be distracting. It'd be one or two lines at most usually.

Reply:
Sure, leave it then.